legacy/builder/constants/constants.go

```diff
@@ -31,7 +31,6 @@ const BUILD_PROPERTIES_COMPILER_WARNING_FLAGS = "compiler.warning_flags"
 const BUILD_PROPERTIES_FQBN = "build.fqbn"
 const BUILD_PROPERTIES_INCLUDES = "includes"
 const BUILD_PROPERTIES_OBJECT_FILE = "object_file"
-const BUILD_PROPERTIES_OBJECT_FILES = "object_files"
```

Comment:
Why replace this constant with the literal string? Is the idea to move away from these (build property) constants? You've also introduced some new literal strings without constants elsewhere.

Reply:
here we go: https://github.com/arduino/arduino-cli/pull/961/commits/d2dfefc11da0d188192e1169e983048eac205cb9